docs/nydus-failover-upgrade.md

```diff
@@ -0,0 +1,68 @@
+The following document has been translated from Chinese to English:
+
+Currently, nydusd supports failover and hot upgrade under fusedev and fscache modes, specifically:
+
+- After a nydusd process crashes, a newly started nydusd process can automatically recover the mount points and other filesystem state information of the crashed nydusd without affecting the normal operation of containers.
```

Comment:
It is better to include the expression: `after nydusd crashes then recovers, it can still serve container I/O operations`.